src/ray/util/filesystem.cc

```diff
@@ -61,13 +61,15 @@ std::string GetUserTempDir() {
 StatusOr<std::string> ReadEntireFile(const std::string &fname) {
   std::ifstream file(fname);
   if (!file.good()) {
-    return Status::IOError("") << "Fails to open file " << fname;
+    return Status::IOError("") << "Fails to open file " << fname << " because "
+                               << strerror(errno);
```

Comment:
```suggestion
    return Status::IOError("") << "Failed to open file " << fname << " because "
                               << strerror(errno);
```

Reply:
updated both